content/cli/overview.md

```diff
@@ -1,23 +1,79 @@
 ### Overview
 
-[Nest CLI](https://github.com/nestjs/nest-cli) is a command-line interface tool that helps you to initialize and develop your applications. It helps on many grounds at once, from scaffolding the project to building well-structured apps. The Nest CLI is based on the [Angular Devkit](https://github.com/angular/devkit) package.
+The [Nest CLI](https://github.com/nestjs/nest-cli) is a command-line interface tool that helps you to initialize, develop, and maintain your Nest applications. It assists in multiple ways, including scaffolding the project, serving it in development mode, and building and bundling the application for production distribution.  It embodies best-practice architectural patterns to encourage well-structured apps.
+
+The Nest CLI is based on the [Angular Devkit](https://github.com/angular/devkit) package, and works for server-side applications in much the same way that the [Angular CLI](https://angular.io/cli) assists development of client-side Angular apps.
```

Comment:
> The Nest CLI is based on the [Angular Devkit](https://github.com/angular/devkit) package

I think that we should remove this sentence. Basically, only `generate` commands are using Angular Devkit underneath, nothing else